docs/install.md

```diff
@@ -0,0 +1,76 @@
+# Install soci-snapshotter
+
+This doc walks through how to install soci-snapshotter as a component managed by systemd.
+
+The soci-snapshotter project produces 2 binaries:
+
+- `soci`: the CLI tool used to build/manage SOCI indices.
+- `soci-snapshotter-grpc`: the daemon (a containerd snapshotter plugin) used for lazy loading.
+
+You can get the prebuilt binaries from our [release page](https://github.com/awslabs/soci-snapshotter/releases)
+or [build them from source](./build.md).
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Dependencies](#dependencies)
+- [Configure soci-snapshotter (optional)](#configure-soci-snapshotter-optional)
+- [Install soci-snpashotter for containerd with systemd](#install-soci-snpashotter-for-containerd-with-systemd)
+- [Config containerd](#config-containerd)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Dependencies
+
+soci-snapshotter has the following dependencies. Please follow the links or commands
+to install them on your machine:
+
+> We only mention the direct dependencies of the project. Some dependencies may
+> have their own dependencies (e.g., containerd depends on runc/cni). Please refer
+> to their doc for a complete installation guide (mainly containerd).
+
+- **[containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) >= 1.4** -
+required to run soci-snapshotter; to confirm please check with `sudo ctr version`.
+- **fuse** - used for mounting without root access (`sudo yum install fuse`).
+
+For fuse/zlib, they can be installed by your Linux package manager (e.g., `yum` or `apt-get`).
+
+## Configure soci-snapshotter (optional)
```

Comment:
We should have a follow up with some documentation on what configuration options are available.

Reply:
agree. added to https://github.com/awslabs/soci-snapshotter/issues/405 - Example config toml with all config options and default values.